.rubocop.yml

```diff
@@ -23,3 +23,6 @@ Sorbet/TrueSigil:
 
 Sorbet/EnforceSigilOrder:
   Enabled: true
+
+Sorbet/ForbidTUntyped:
+  Enabled: false
```

Comment:
Should I disable this or fix all the `T.untyped` entries in the project?

Reply:
I'm surprised it is enabled by default since we didn't put it in the [default config](https://github.com/Shopify/rubocop-sorbet/blob/main/config/default.yml)? If it's the case I think we should disable it by default as it's going to be painful for a lot of users.

Reply:
It is because we have `Sorbet: Enabled: true` here. I think if you don't have that it doesn't enable by default. But I can add it to the default.yml as disabled.

Reply:
Actually, it is already there as disabled by default. I'm not confused as well.

Reply:
Ah, it is the `T.unsafe` that is there. Let me add